_data/contributors.yml

```diff
@@ -214,6 +214,29 @@
   email: ioana.ifrim@cern.ch
   education: MPhil Advanced Computer Science, University of Cambridge (2018)
 
+- name: Isaac Morales Santana
```

Comment:
This entry needs to move to the list of active contributors. You added it to the section of the retired ones. Maybe move it higher in the file.